lightly/models/modules/nn_memory_bank.py

```diff
@@ -0,0 +1,62 @@
+""" Nearest Neighbour Memory Bank Module """
+
+# Copyright (c) 2021. Lightly AG and its affiliates.
+# All Rights Reserved
+
+import torch
+from lightly.loss.memory_bank import MemoryBankModule
+
+
+class NNmemoryBankModule(MemoryBankModule):
```

Comment:
It's a little off-looking to me that it's called "NNmemoryBank" instead of "NNMemoryBank" but we can go with it if you think yours is better.

Reply:
I agree. I'll change that one. `NNMemoryBank` it is :)